packages/next/src/build/webpack/plugins/flight-manifest-plugin.ts

```diff
@@ -136,8 +136,8 @@ export class ClientReferenceManifestPlugin {
 
     // Collect client requests
     function collectClientRequest(mod: webpack.NormalModule) {
-      if (mod.resource === '' && mod.buildInfo.rsc) {
-        const { requests = [] } = mod.buildInfo.rsc
+      if (mod.resource === '' && mod.buildInfo?.rsc) {
+        const { requests = [] } = mod.buildInfo?.rsc
```

Comment:
```suggestion
        const { requests = [] } = mod.buildInfo.rsc
```

I think this line can remain the same since the `if` condition makes sure it exists